orphedomos-maven-plugin/pom.xml

```diff
@@ -5,7 +5,7 @@
     <parent>
         <groupId>org.technologybrewery.orphedomos</groupId>
         <artifactId>orphedomos-parent</artifactId>
```

Comment:
C (Critical): there are some version mismatches.  I think we probably need to get these all on the 0.11.0-SNAPSHOT version, correct?

If so, that can be done w/ the the `versions` plugin: https://www.mojohaus.org/versions/versions-maven-plugin/set-mojo.html

Reply:
Looks right to me!